### PR TITLE
Add meeting notes for May 30 and June 20, 2019

### PR DESCRIPTION
**Pull request by jlpalomino**
@lwasser @choldgraf This PR adds the PyOpenSci meeting notes for May 30th and June 20th, 2019.

### CODE REVIEW

File: meeting-notes/2019-06-20-notes.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+* Create a shared hackmd and post it on the discourse forum. 
+    * for onboarding -- we want name, email, github username, what they work on (what domain they are in) what they may want to contibute??
+    * Joss has a google form that they can submit if they want to be a reviewer.
+        * [JOSS reviewer list](https://docs.google.com/spreadsheets/d/1PAPRJ63yq9aPC1COLjaQp8mHmEq3rZUzwUYxTulyu78/edit?usp=sharing)
+    * How does rOpenSci keep track of / pick / rotate reviewers?
+    * joss has a dashboard to keep track of reviewers / reviews. might be worth chatting with Arfon about how that works. 
+
+* If you will be at SciPy, please add your name below:
+    * Leah
+    * Jenny
```
Comment (choldgraf) [inline meeting-notes/2019-06-20-notes.md]:
```suggestion
    * Jenny
    * Chris H
```

File: meeting-notes/2019-06-20-notes.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+5. Roles reminder (we can skip this if there's not time today)
+
+## Role Definition Revisited (please feel free to edit this or leave comments!!)
+
+Starting with the [rOpenSci structure](https://ropensci.org/about/) but we may want to adjust accordingly. And ofcourse the time commitment can really be flexible but just trying to give folks a sense of time that might be associated with each role. 
+
+## Leadership (3-5): 
+* Responsible for making final decisions about the direction of POS
+* POS
+* Communication: weekly / bi weekly, slack, discuss private forum (Chris H can help us set this up i think??, emails)
```
Comment (choldgraf) [inline meeting-notes/2019-06-20-notes.md]: lemme know if you'd like me to do something here

File: meeting-notes/2019-06-20-notes.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+NOTES: JOSS model: Joss has a small number (4) of editors in chief. each week one is on call and responsible for everything. has a larger pool of associated editors. this might work better if there is a larger community who want to jump in here and there. 
+
+## Advisory (4*):
+### Responsibilities:
+* Provide feedback & guidance regarding the direction of POS
+* Spread the word about POS
+* Communication: every other month / quarterly? meetings??
+---
+* Carson  (external/industry advisor)
+* Neil
```
Comment (choldgraf) [inline meeting-notes/2019-06-20-notes.md]: right now this is probably the role that I can most-confidently commit to

**Comment by choldgraf** [comment]
looks great to me! merging away

File: meeting-notes/2019-06-20-notes.md (hunk 3 above)
Comment (lwasser) [reply meeting-notes/2019-06-20-notes.md]: oh @choldgraf feel free to add your name here. we can create a running list of roles somewhere more visible as well if we want!

File: meeting-notes/2019-06-20-notes.md (hunk 2 above)
Comment (lwasser) [reply meeting-notes/2019-06-20-notes.md]: oh - perhaps what would be good is just telling people about the various forums in discourse as you did set this up. we aren't using it yet however.

**Pull request merged**